src/components/Fields/Examples.tsx

```diff
@@ -31,6 +31,5 @@ export function Examples({ field }: { field: FieldModel }) {
 
 const ExamplesList = styled.ul`
   margin-top: 1em;
-  padding-left: 0;
-  list-style-position: inside;
+  padding-left: 17px;
```

Comment:
Is it possible to use here something other than `17px;`? Like something from the theme or doesn't use at all

Reply:
I can use  `list-style-position: outside;` instead of `padding-left: 17px; `
and it will look like this
![Screenshot 2022-04-29 at 11 16 12](https://user-images.githubusercontent.com/28001084/165908343-64e30648-a699-4782-ac43-0ca73326a929.png)